py-polars/polars/lazyframe/frame.py

```diff
@@ -1812,6 +1812,33 @@ def collect_async(
         ldf.collect_with_callback(result._callback)
         return result
 
+    def _sink_generic(
```

Comment:
This is not a sink method - it's a method to set the optimization flags for sink methods. So it needs a better name.

```suggestion
    def _set_sink_optimizations(
```

I would also prefer this to be defined below the sink methods (newspaper style).

Reply:
Thanks. Hardest for a dev: naming stuff...